app/components/primer/details_component.rb

```diff
@@ -1,11 +1,7 @@
 # frozen_string_literal: true
 
 module Primer
-  #
-  # overlay - options are `none`, `default` and `dark`. Dictates the type of overlay to render with.
-  # button - options are `default` and `reset`. default will make the target a default primer ``.btn`
-  #          reset will remove all styles from the <summary> element.
-  #
+  # Details classes are created to enhance the native behaviors of the details element
```

Comment:
How about:

```suggestion
  # Use DetailsComponent to reveal content after clicking a button.
```

Or something similar?